internal/scorecard/alpha/formatting.go

```diff
@@ -60,11 +60,11 @@ func (o Scorecard) ListTests() (output v1alpha2.ScorecardOutput, err error) {
 	return output, err
 }
 
-func testResultError(err error, test Test) v1alpha2.ScorecardTestResult {
+func testResultError(err error, test Test) *v1alpha2.ScorecardTestResult {
 	r := v1alpha2.ScorecardTestResult{}
 	r.Name = test.Name
 	r.State = v1alpha2.FailState
 	r.Description = test.Description
 	r.Errors = []string{err.Error()}
-	return r
+	return &r
 }
```

Comment:
not a blocker .. but would be nice cover this one as well with tests. 
See; https://coveralls.io/builds/30552210/source?filename=internal/scorecard/alpha/formatting.go

Reply:
thanks, added a new unit test, formatting_test.go.